documentation/modules/collected-logs-cr-info.adoc

```diff
@@ -18,14 +18,15 @@ The `must-gather` tool collects the following logs and CR files in an archive fi
 ** `DataVolume` CR: Represents a disk mounted on a migrated VM.
 ** `VirtualMachine` CR: Represents a migrated VM.
 ** `Plan` CR: Defines the VMs and storage and network mapping.
+** `Job` CR: Optional: Represents pre-migration or post-migration hook.
```

Comment:
I guess `and/or` should be here, cause both pre- and after-hook can be present in the same plan.